docs/education/case_studies.md

```diff
@@ -16,7 +16,7 @@ A license is no longer required to use Clara Parabricks 4.x and later versions,
 
 1. Access to internet.
 
-2. Any GPU that supports CUDA architecture 60, 61, 70, 75 and has 12GB GPU RAM or more. It has been tested on NVIDIA V100, NVIDIA A100, and NVIDIA T4 GPUs.
+2. Any GPU that supports CUDA architecture 6.0, 6.1, 7.0, 7.5 and has 12GB GPU RAM or more. It has been tested on NVIDIA V100, NVIDIA A100, and NVIDIA T4 GPUs.
```

Comment:
Could you update the CUDA architecture, GPU RAM, and the test platform? 
I found the recent document has this update: https://docs.nvidia.com/clara/parabricks/4.1.0/gettingstarted.html.

Reply:
Will do, thank you!

Reply:
From our discussion it sounds like they've dropped official support for P100s, but they do work with Parabricks 4.0.0. We'll want to test with 4.1.0 as well.

Applies to #614 as well.

Reply:
Thanks for noting! 
The version tested was 4.0.0. Here is the link to documentation : https://docs.nvidia.com/clara/parabricks/4.0.0/gettingstarted.html. Thanks for notin

Reply:
We're going to leave the docs at 4.0.0 for now, and update to 4.1.0 at a later time when we've had a chance to test.